src/backend/base/langflow/custom/custom_component/component.py

```diff
@@ -1137,6 +1145,43 @@ def send_error(
 
     def _append_tool_to_outputs_map(self):
         self._outputs_map[TOOL_OUTPUT_NAME] = self._build_tool_output()
+        # add a new input for the tool schema
+        # self.inputs.append(self._build_tool_schema())
 
     def _build_tool_output(self) -> Output:
         return Output(name=TOOL_OUTPUT_NAME, display_name=TOOL_OUTPUT_DISPLAY_NAME, method="to_toolkit", types=["Tool"])
+
+    def _build_tools_metadata_input(self):
+        tools = self.to_toolkit()
+        table_schema = [
+            {
+                "name": "name",
+                "display_name": "Name",
+                "type": "str",
+                "description": "Specify the name of the output field.",
+            },
+            {
+                "name": "description",
+                "display_name": "Description",
+                "type": "str",
+                "description": "Describe the purpose of the output field.",
```

Comment:
what do you thing if we create consts for those hardCoded values in description?